### PR TITLE
Cloud benchmarking

### LINKED ISSUE #18: Cloud benchmarking
Prepare and execute a benchmark plan on different clouds to test performance in different providers and VM flavors.

### PR DESCRIPTION
**Pull request by suizman**
Closes #18

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 *.dll
 *.so
 *.dylib
+qed
```
Comment (iknite) [inline .gitignore]: Why? this proyect doesn't have that folder, so it doesn't need to be ignored (As far as I know)

File: tests/cloud/aws/variables.tf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+// Force choose instance size.
+variable "flavour" {}
```
Comment (iknite) [inline tests/cloud/aws/variables.tf]: Do we need defaults?

File: .gitignore (hunk 1 above)
Comment (suizman) [reply .gitignore]: It prevents accidentally upload to Github the QED binary automatically generated by the main.tf in order to provision the AWS instances with it.

File: tests/cloud/aws/variables.tf (hunk 2 above)
Comment (suizman) [reply tests/cloud/aws/variables.tf]: Maybe for us is useful to have a default value but for other people who wants to perform benchmark on AWS bringing an c5.2xlarge instance by default might be an issue.
We can discuss about providing an alternative variables file.

**Comment by suizman** [comment]
Thanks guys :ok_hand:

**Pull request merged**